src/groups/mqb/mqbi/mqbi_storagemanager.h

```diff
@@ -217,6 +221,10 @@ class StorageManager : public mqbi::AppKeyGenerator {
     /// Stop this storage manager.
     virtual void stop() = 0;
 
+    /// Initialize the queue key info map based on information in the specified
+    /// `clusterState`.
```

Comment:
With regard to the comment above, I think adding "The behavior of this function is undefined unless the queue key info map has not yet been initialized."  Doesn't mean we need to `ASSERT` inside, logging or even doing nothing is sufficient.

Reply:
Per above, there are legal ways to call it more than once. Therefore, I plan to update the comment to say `Note that this method should only be called once; subsequent calls will be ignored.`

Reply:
^ this makes sense to me.